lib/gcf_sms/gcf_sms.c

```diff
@@ -283,14 +283,14 @@ static int at_cmd_callback_csca(char *buf, size_t len, char *at_cmd)
 {
 	/* Set */
 	if (prefix_match("AT+CSCA=", at_cmd)) {
-		strncpy(sca_buff, &(at_cmd[8]), SCA_SIZE_MAX);
+		strncpy(sca_buff, &(at_cmd[8]), CSCA_SIZE_MAX);
 		sca_buff[sizeof(sca_buff) - 1] = '\0';
 		return at_cmd_custom_respond(buf, len, "OK\r\n");
 	}
 
 	/* Read */
 	if (prefix_match("AT+CSCA?", at_cmd)) {
-		return at_cmd_custom_respond(buf, len, "+CSMS: %s\r\nOK\r\n", sca_buff);
+		return at_cmd_custom_respond(buf, len, "+CSCA: %s\r\nOK\r\n", sca_buff);
```

Comment:
@eivindj-nordic, was this intentionally `+CSMS` and if yes, why? Tests have the same so I'm starting to suspect that this is not a copy-paste bug.

Reply:
Hi, it was intentional, I'll send you the (internal) reference.

Reply:
Seems this was not intentional and I'm discussing what fixes we do.

Reply:
I will close this comment as we have decided to fix the typo.